packages/webpack/test/e2e-main.spec.js

```diff
@@ -0,0 +1,75 @@
+const test = require('ava')
+const { scaffold, runScriptWithSES } = require('./scaffold.js')
+const webpackConfigDefault = require('./fixtures/main/webpack.config.js')
+
+test.before(async (t) => {
+  const webpackConfig = { ...webpackConfigDefault }
+  await t.notThrowsAsync(async () => {
+    t.context.build = await scaffold(webpackConfig)
+  }, 'Expected the build to succeed')
+  t.context.bundle = t.context.build.snapshot['/dist/app.js']
```

Comment:
a reasonable thing instead would be to assert that this value exists

Reply:
oh also, see `/scripts/publish.spec.js` for how to properly type a test execution context, if you care